### PR TITLE
Fixing token expiration length

### PR DESCRIPTION
**Pull request by jspaith**

### CODE REVIEW

**Comment by jspaith** [comment]
@jebrando is added to the review. #Closed

File: iothub_client/src/iothub_client_authorization.c [hunk 1]
```diff
@@ -314,7 +314,7 @@ char* IoTHubClient_Auth_Get_SasToken(IOTHUB_AUTHORIZATION_HANDLE handle, const c
             }
             else 
             {
-                size_t expiry_time = sec_since_epoch+expire_time;
+                size_t expiry_time = sec_since_epoch+expiry_time_relative_seconds;
```
Comment (anthonyvercolano) [inline iothub_client/src/iothub_client_authorization.c]: >+expiry_time_relative_seconds; [](start = 52, length = 30)

Not your fault but I find it very irritating that we are basically doing this computation twice in this function.

**Comment by jspaith** [comment]
@jebrando is added to the review. #Closed

**Comment by jspaith** [comment]
@ewertons is added to the review. #Closed

**Comment by jspaith** [comment]
@jspaith is added to the review. #Closed

File: iothub_client/tests/iothubtransport_mqtt_common_ut/iothubtransport_mqtt_common_ut.c [hunk 2]
```diff
@@ -276,11 +276,11 @@ extern "C"
 }
 #endif
 
-char* my_IoTHubClient_Auth_Get_SasToken(IOTHUB_AUTHORIZATION_HANDLE handle, const char* scope, size_t expire_time)
+char* my_IoTHubClient_Auth_Get_SasToken(IOTHUB_AUTHORIZATION_HANDLE handle, const char* scope, size_t expiry_time_relative_seconds)
```
Comment (jebrando) [inline iothub_client/tests/iothubtransport_mqtt_common_ut/iothubtransport_mqtt_common_ut.c]: static my_IoTHu...

File: iothub_client/tests/iothubtransport_amqp_cbs_auth_ut/iothubtransport_amqp_cbs_auth_ut.c [hunk 3]
```diff
@@ -154,11 +154,11 @@ static int TEST_cbs_put_token_async(CBS_HANDLE cbs, const char* type, const char
     return TEST_cbs_put_token_async_return;
 }
 
-char* TEST_IoTHubClient_Auth_Get_SasToken(IOTHUB_AUTHORIZATION_HANDLE handle, const char* scope, size_t expire_time)
+char* TEST_IoTHubClient_Auth_Get_SasToken(IOTHUB_AUTHORIZATION_HANDLE handle, const char* scope, size_t expiry_time_relative_seconds)
```
Comment (jebrando) [inline iothub_client/tests/iothubtransport_amqp_cbs_auth_ut/iothubtransport_amqp_cbs_auth_ut.c]: static char* TEST_IoTH...

**Pull request merged**